project/settings_hls.tcl

```diff
@@ -12,17 +12,17 @@ config_compile -name_max_length 100
 # Set clock frequency
 create_clock -period 250MHz -name default
 
-# Set FPGA variable with part number
```

Comment:
maybe this is just WIP but please don't commit this change as we want to move to the VU13P ...

Reply:
@kwalkingshaw Can you revert this file to what's on the master branch? The FPGA part number should now be set only in project/fpga.tcl.